packages/base-driver/lib/express/server.js

```diff
@@ -208,23 +204,34 @@ function configureHttp({httpServer, reject, keepAliveTimeout}) {
 
   // http.Server.close() only stops new connections, but we need to wait until
   // all connections are closed and the `close` event is emitted
-  const close = appiumServer.close.bind(appiumServer);
+  const originalClose = appiumServer.close.bind(appiumServer);
   appiumServer.close = async () =>
-    await new B((resolve, reject) => {
-      // https://github.com/nodejs/node-v0.x-archive/issues/9066#issuecomment-124210576
-      serverState.closed = true;
-      serverState.notifier.emit('shutdown');
-      log.info('Waiting until the server is closed');
-      httpServer.on('close', () => {
-        log.info('Received server close event');
-        resolve();
+    await new B((_resolve, _reject) => {
```

Comment:
Should these parameters be underscored?

Reply:
yes , to avoid them shadowing the top function arguments